doc/user_guide/interactions.rst

```diff
@@ -784,36 +791,6 @@ where the user can choose the colors of the chart interactively:
         color_usa, color_europe, color_japan
     )
 
-.. _encoding-channel-binding:
-
-Encoding Channel Binding
-^^^^^^^^^^^^^^^^^^^^^^^^
-
-There is no direct way to map an encoding channel to a widget in order to dynamically display different charts based on different column choices, such as ``y=column_param``. The underlying reason this is not possible is that in Vega-Lite, the ``field`` property does not accept a parameter as value; see the `field Vega-Lite documentation <https://vega.github.io/vega-lite/docs/field.html>`_. You can follow the discussion in this issue https://github.com/vega/vega-lite/issues/7365, and in the meantime, you can use parameters for a convenient workaround which let's you achieve the same functionality and change the plotted columns based on a widget selection (the x-axis title cannot be changed dynamically, but a text mark could be used instead if desired):
-
-.. altair-plot::
-
-    dropdown = alt.binding_select(
-        options=['Horsepower', 'Displacement', 'Weight_in_lbs', 'Acceleration'],
-        name='X-axis column '
-    )
-    xcol_param = alt.param(
-        value='Horsepower',
-        bind=dropdown
-    )
-
-    alt.Chart(data.cars.url).mark_circle().encode(
-        x=alt.X('x:Q').title(''),
-        y='Miles_per_Gallon:Q',
-        color='Origin:N'
-    ).transform_calculate(
-        x=f'datum[{xcol_param.name}]'
-    ).add_params(
-        xcol_param
-    )
-
-It was possible to achieve something similar before the introduction of parameters in Altair 5 by using ``transform_fold`` and ``transform_filter``, but the spec for this is more complex (as can be seen in `this SO answer <https://stackoverflow.com/a/70950329/2166823>`_) so the solution above is to prefer.
```

Comment:
I didn't include this section in this PR because I think it's not really relevant anymore now that Altair 5 has been out for a while and using params is so much simpler.